ftw_tools/models/baseline_inference.py

```diff
@@ -121,12 +153,12 @@ def run(
         dataset,
         sampler=sampler,
         batch_size=batch_size,
-        num_workers=6,
+        num_workers=num_workers,
```

Comment:
I removed the hardcoding of num_workers here to make it configurable from the CLI